remote/remote.go

```diff
@@ -237,7 +237,7 @@ func authConn(conn net.Conn) {
 	wpCLIProcess, found := gGUIDttys[GUID]
 	padlock.Unlock()
 
-	if found && wpCLIProcess.Running {
+	if found {
```

Comment:
Great question.

Actually, if the command is `cancelled` it will be blocked by the API, since it will be marked as cancelled on the API and API will not allow it to be resumed.